src/DrDotnet.Profilers/profilers/allocation_by_class_profiler/mod.rs

```diff
@@ -1,8 +1,8 @@
 use dashmap::DashMap;
-use profiling_api::*;
 use uuid::Uuid;
 use std::sync::atomic::{AtomicIsize, Ordering};
 
```

Comment:
In the beginning, I had one file per profiler but then I made folders to put README (documentation) next to it however I am kind of backpedaling on this idea I am thinking it's better to put all documentation-related stuff in a `docs` directory at the repo root because we won't just document profilers but also DrDotnet in general, and there will be pictures and other documentation material.   
So yeah good point I'll do that :)

Reply:
done